src/poll.js

```diff
@@ -254,7 +254,7 @@ class Poll {
       if (answer.people.length > 0) {
         msg
           .attachment()
-          .text(`*${answer.people.length}*  ${answer.text} » ${answer.people.join(',')}`)
+          .text(`*${answer.people.length}*  ${answer.text}  ⇢  _${answer.people.map((it) => `@${it}`).join(', ')}_`)
```

Comment:
LGTM, small nit/fyi - if your fat arrow function only has one param, you can leave off the parens.  If you already knew this, please disregard 😄 :

```
answer.people.map(it => `@${it}`)
```